notebooks/official/feature_store/sdk-feature-store-pandas.ipynb

```diff
@@ -30,15 +30,23 @@
       },
```

Comment:
<p>You are charged for Vertex AI usage. For more information, see <a href="https://cloud.google.com/vertex-ai/pricing" target="_blank">Vertex AI pricing</a>. You can also use the <a href="https://cloud.google.com/products/calculator/" target="_blank">Pricing Calculator</a>&nbsp;to estimate costs based on your projected usage.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>